src/main/java/io/swagger/oas/inflector/validators/NumericValidator.java

```diff
@@ -101,11 +103,11 @@ public void validate(Object o, RequestBody body, Iterator<Validator> chain) thro
                 if (body.getContent().get(media) != null) {
                     MediaType mediaType = body.getContent().get(media);
                     if (o != null && mediaType.getSchema() != null) {
```

Comment:
that "o" is everywhere but what is it? o.O

Reply:
done